_clients/data-prepper/get-started.md

```diff
@@ -61,3 +88,52 @@ After starting Data Prepper, you should see log output and some UUIDs after a fe
 e51e700e-5cab-4f6d-879a-1c3235a77d18
 b4ed2d7e-cf9c-4e9d-967c-b18e8af35c90
 ```
+The remainder of this page shows examples for running from the Docker image. If you
+built from source, refer to [Developer Guide](https://github.com/opensearch-project/data-prepper/blob/main/docs/developer_guide.md) for more information.
+
+However you configure your pipeline, you will run Data Prepper the same way. You run the Docker
+image and supply both the `pipelines.yaml` and `data-prepper-config.yaml` files.
+
+For Data Prepper 2.0 or above, use this command:
+
+```
+docker run --name data-prepper -p 4900:4900 -v ${PWD}/pipelines.yaml:/usr/share/data-prepper/pipelines/pipelines.yaml -v ${PWD}/data-prepper-config.yaml:/usr/share/data-prepper/config/data-prepper-config.yaml opensearchproject/data-prepper:latest
+```
+
+For Data Prepper before version 2.0:
+
+```
+docker run --name data-prepper -p 4900:4900 -v ${PWD}/pipelines.yaml:/usr/share/data-prepper/pipelines.yaml -v ${PWD}/data-prepper-config.yaml:/usr/share/data-prepper/data-prepper-config.yaml opensearchproject/data-prepper:1.x
+```
+
+Once Data Prepper is running, it will process data until it is shutdown. Once you are done, shut it down with
+
+```
+curl -X POST http://localhost:4900/shutdown
+```
+### Additional Configurations
+
+For Data Prepper 2.0 or above, Log4j 2 configuration file is read from `config/log4j2.properties` in the application's home directory. 
+By default, it's using `log4j2-rolling.properties` in the *shared-config* directory.
+
+For Data Prepper before version 2.0, optionally add `"-Dlog4j.configurationFile=config/log4j2.properties"` to the command if you would 
+like to pass a custom log4j2 properties file. If no properties file is provided, Data Prepper will default to the log4j2.properties file in the *shared-config* directory.
+
+## Next Steps
+
+All Data Prepper instances expose a few APIs. The [API documentation]({{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/api/) outlines these APIs and
+how to configure the server.
+
+Trace Analytics is an important Data Prepper use case. If you haven't yet configure it,
+please visit the [Trace Analytics documentation]{{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/trace-analytics/).
+
+Log Ingestion is also an important Data Prepper use case. To learn more, visit the [Log Ingestion Documentation]({{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/log-analytics/).
+
+To run Data Prepper with a Logstash configuration, please visit the [Logstash Migration Guide]({{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/logstash-migration-guide/).
+
+To monitor Data Prepper, please read the [Monitoring]({{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/monitoring/) page.
+
+## Other Examples
+
+We have other several Docker [examples](https://github.com/opensearch-project/data-prepper/tree/main/examples/)
+that allow you to run Data Prepper in different scenarios.
```

Comment:
I would rephrase this as "For other Docker examples that allow you to run Data Prepper in different scenarios, see [link]."